server/controllers/finance/fiscal.js

```diff
@@ -648,3 +649,25 @@ function getPeriodByFiscal(fiscalYearId) {
 
   return db.exec(sql, [fiscalYearId]);
 }
+
+/**
+ * @method lookupFiscalYearByDate
+ *
+ * @description
+ * This function returns a single record from the fiscal year table matching
+ * the DATE provided.  If no record is found, it throws a NotFound error.
```

Comment:
I don't think this function throws a `NotFound` error.  If you used `db.one()` it would.